<launch>
     
    <rosparam command="load" file="$(find last_letter_2)/param/HID.yaml" />
    <node pkg="joy" type="joy_node" name="joystick" output="screen" >
      <param name="dev" value="/dev/input/js0" />
      <param name="deadzone" value="0.01" />
    </node>
    <node pkg="last_letter_2" name="joy2chan" type="joy2chan" output="screen" />
    <!-- <node pkg="last_letter_2" name="chan2basics" type="chan2basics" output="screen"/> -->
    <!-- <node pkg="last_letter_2" name="controller_node" type="controller_node" output="screen" /> -->
    <!-- <node pkg="last_letter_2" name="master" type="master" output="screen" /> -->
    
</launch>
    
    